src/api/users/users.services.ts

```diff
@@ -48,9 +56,30 @@ export const createUser = (input: any) => {
 
 // update user
 export const updateUser = (id: string, input: any) => {
+  const { 
+    email,
+    password, 
+    name, 
+    last_name, 
+    city, 
+    contact_email, 
+    contact_sms, 
+    contact_wpp,
+    phone_numbers, 
+    addresses
+   } = input;
 
-  const { email, password, name, last_name, city, contact_email, contact_sms, contact_wpp } = input;
+  // Update the phone numbers
+  const updatedPhoneNumbers = phone_numbers.map((phone_number: any) => ({
+    where: { id_user_phone_number: phone_number.id_user_phone_number },
+    data: { phone_number: phone_number.phone_number },
+  }));
 
+  // Update the addresses
+  const updatedAddresses = addresses.map((address: any) => ({
+    where: { id_address: address.id_address },
+    data: { address_name: address.address_name, address: address.address, city: address.city },
```

Comment:
why not destructure the item object in map for both updating processes? would be better yet, but then, I approved, LGTM.

Reply:
resolved